package.json

```diff
@@ -1,225 +1,226 @@
+        "@babel/preset-env": "^7.11.0",
+        "@babel/preset-flow": "^7.12.13",
+        "@babel/preset-react": "^7.10.4",
+        "@babel/runtime": "^7.11.2",
+        "@octokit/core": "4.0.4",
+        "@octokit/plugin-paginate-rest": "3.1.0",
+        "@octokit/plugin-throttling": "4.1.0",
+        "@react-native-community/eslint-config": "3.0.0",
+        "@react-navigation/devtools": "^6.0.10",
+        "@storybook/addon-a11y": "^6.5.9",
+        "@storybook/addon-essentials": "^6.5.9",
+        "@storybook/addon-react-native-web": "0.0.19--canary.37.cb55428.0",
+        "@storybook/addons": "^6.5.9",
+        "@storybook/builder-webpack5": "^6.5.10",
+        "@storybook/manager-webpack5": "^6.5.10",
+        "@storybook/react": "^6.5.9",
+        "@storybook/theming": "^6.5.9",
+        "@svgr/webpack": "^6.0.0",
+        "@testing-library/jest-native": "5.4.1",
+        "@testing-library/react-native": "11.5.1",
+        "@vercel/ncc": "^0.27.0",
+        "@welldone-software/why-did-you-render": "7.0.1",
+        "ajv-cli": "^5.0.0",
+        "babel-eslint": "^10.1.0",
+        "babel-jest": "29.4.1",
+        "babel-loader": "^8.1.0",
+        "babel-plugin-module-resolver": "^5.0.0",
+        "babel-plugin-react-native-web": "^0.18.7",
+        "babel-plugin-transform-class-properties": "^6.24.1",
+        "babel-plugin-transform-remove-console": "^6.9.4",
+        "clean-webpack-plugin": "^3.0.0",
+        "concurrently": "^5.3.0",
+        "copy-webpack-plugin": "^6.4.1",
+        "css-loader": "^6.7.2",
+        "diff-so-fancy": "^1.3.0",
+        "dotenv": "^16.0.3",
+        "electron": "22.3.7",
+        "electron-builder": "23.5.0",
+        "electron-notarize": "^1.2.1",
+        "eslint": "^7.6.0",
+        "eslint-config-expensify": "^2.0.36",
+        "eslint-config-prettier": "^8.8.0",
+        "eslint-plugin-jest": "^24.1.0",
+        "eslint-plugin-jsx-a11y": "^6.6.1",
+        "eslint-plugin-react-hooks": "^4.6.0",
+        "eslint-plugin-storybook": "^0.5.13",
+        "flipper-plugin-bridgespy-client": "^0.1.9",
+        "html-inline-script-webpack-plugin": "^3.1.0",
+        "html-webpack-plugin": "^5.5.0",
+        "jest": "29.4.1",
+        "jest-circus": "29.4.1",
+        "jest-cli": "29.4.1",
+        "jest-environment-jsdom": "^29.4.1",
+        "metro-react-native-babel-preset": "^0.73.3",
+        "mock-fs": "^4.13.0",
+        "onchange": "^7.1.0",
+        "portfinder": "^1.0.28",
+        "prettier": "^2.8.8",
+        "pusher-js-mock": "^0.3.3",
+        "react-native-clean-project": "^4.0.0-alpha4.0",
+        "react-native-flipper": "https://gitpkg.now.sh/facebook/flipper/react-native/react-native-flipper?9cacc9b59402550eae866e0e81e5f0c2f8203e6b",
+        "react-native-performance-flipper-reporter": "^2.0.0",
+        "react-native-svg-transformer": "^1.0.0",
+        "react-test-renderer": "18.1.0",
+        "setimmediate": "^1.0.5",
```

Comment:
and setimmediate as a dev dependency since it used to be part of reanimated and is still used in tests right now